### PR TITLE
Reorganize and split into separate runtime components

### LINKED ISSUE #24: Create separate images for each runtime
The .NET runtime is rather sizeable (140MB) and downloading and installing the duct-tape image imposes a noticeable lag on container startup.

### PR DESCRIPTION
**Pull request by briandealwis**
Split out the debug support runtime container image into separate images for each runtime component.  Necessary as the .NET Core debug support files weigh in at a hefty 190MB.

Fixes #24

### CODE REVIEW

**Review by loosebazooka — APPROVED** [review]
This looks fine, I do think editor/github interactions for Dockerfile could benefit from doing `go/Dockerfile` vs `Dockerfile.go`? The syntax highlighting is super weird now, and I image intellij/vscode might suffer from the same fate?

It would also affect the skaffold.yaml.

It's kind of up to you though. I could see this submitted as is, but lean towards separate directories.

EDIT: I said all this before I realized there was a shared `install.sh`... Now I'm less sure which side I lean on.

**Comment by briandealwis** [comment]
Good point on the extensions — it is a bit confusing.  I flattened out the directories to share the install.sh, but that's such a tiny script that duplicating it isn't the end of the world.

**Comment by briandealwis** [comment]
Pulled in the Travis settings to test that the minor reorganization changes do what they're supposed to.

**Review by loosebazooka — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+make available any support files required to debug specific language
+runtimes.  For example, the `k8s-pod.yaml.orig` file shows an k8s
+podspec that would be transformed to `k8s-pod.yaml` to:
+  - mount a volume to hold the debugging support files
+  - provide an initContainer to populate the volume
+  - mount the volume to the applicable containers
+
+This directory includes a `skaffold.yaml` for development of the
+the `duct-tape` initContainer image.  To add support for a new
+language runtime, run `skaffold dev` and tweak `duct-tape/Dockerfile`
```
Comment (quoctruong) [inline README.md]: The locations in these comments are now invalid after the most recent commit.

Comment (briandealwis) [reply README.md]: Oh good point.

**Pull request merged**